api-docs/leads.md

```diff
@@ -86,152 +86,43 @@ If there is no lead with the provided id, you will get a 404 HTTP response, with
 
 ## Get all Leads
 
-### Request
+## Request
```

Comment:
The Request is a sub header of the Get all Leads header and should have one more level of heading because of that.

```suggestion
### Request
```